pkgs/by-name/ne/newsraft/package.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, stdenv
+, fetchFromGitea
+, pkg-config
+, curl, expat, gumbo, ncurses, sqlite, yajl
```

Comment:
```suggestion
, curl
, expat
, gumbo
, ncurses
, sqlite
, yajl
```

Reply:
These are minor nitpicks, but I feel you should know the general style of nixpkgs contributions, considering this is your first pr.